mlir/include/mlir/Dialect/Tosa/IR/TosaUtilOps.td

```diff
@@ -86,12 +86,13 @@ def Tosa_VariableOp : Tosa_Op<"variable", []> {
   let summary = "Defines a variable";
 
   let description = [{
-    Defines a new TOSA variable. This is a mutable value.
+    Defines a new TOSA variable.
+    This is a persistent mutable value across multiple TOSA graph invocations.
     Modifications are expressed using read/write semantics.
   }];
 
   let arguments = (ins
-    SymbolNameAttr:$name,
+    SymbolNameAttr:$uid,
```

Comment:
I wonder if it would be worth leaving a comment here to explain why there's divergence from the spec?

Also, since we're choosing to use a name rather than uid, I'm wondering if "name" was actually better name for the attribute (apologies for the back-and-forth on this, would be good to see what @GeorgeARM thinks before making the change)?

Reply:
I agree. I will hold merging this.

Reply:
Agreed. Let's keep `name`. Probably worth amending this change and proceed to the rest of the small changes that this patch fixes.